### ISSUE TITLE
4. Usability Issues

### ISSUE THREAD

**Issue by itis-maj** [itis-maj/Requirements-Specification#4]
4. Usability Issues
Complex User Interface: Non-intuitive design may make the system difficult to use.

Lack of Accessibility: The system may not support users with disabilities.

Poor Mobile Experience: Interfaces not optimized for mobile devices hinder usability.
